### PR TITLE
Adding a couple of methods required but auth tests.

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java [hunk 1]
```diff
@@ -318,6 +337,28 @@ protected ClientConfig getClientConfig() {
     return builder.build();
   }
 
+  protected ClientConfig getClientConfig(@Nullable AccessToken accessToken) {
```
Comment (anwar6953) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java:309]: should remove a lot of the duplicate code from the versions of these methods that are parameter-less

File: cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java [hunk 2]
```diff
@@ -216,25 +216,30 @@ public Boolean call() throws Exception {
...
     Properties properties = new Properties();
-    properties.setProperty("security.auth.client.username", System.getProperty("cdap.username"));
-    properties.setProperty("security.auth.client.password", System.getProperty("cdap.password"));
-    final AuthenticationClient authClient = new BasicAuthenticationClient();
+    properties.setProperty("security.auth.client.username", username);
+    properties.setProperty("security.auth.client.password", password);
+    final BasicAuthenticationClient authClient = new BasicAuthenticationClient();
     authClient.configure(properties);
-    ConnectionConfig connectionConfig = getClientConfig().getConnectionConfig();
+    ConnectionConfig connectionConfig = getClientConfig(null).getConnectionConfig();
```
Comment (anwar6953) [inline cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java]: we dont need to pass in null, right?

**Comment by anwar6953** [comment]
LGTM

**Comment by Nishith** [comment]
Green build: http://builds.cask.co/browse/CDAP-DUT5134-1

**Pull request merged**